content/en/docs/reference/capabilities/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Capabilities
+description: Capabilities required to run Falco
+weight: 20
+---
+
+To analyze kernel events we have [3 possible sources](/docs/event-sources/kernel/): kernel module, eBPF probe, modern eBPF probe.
+
+The kernel module can run only in privileged mode while the 2 eBPF probes could also use capabilities to run with fewer privileges.
+
+## eBPF probe
+
+The minimal set of capabilities required by Falco to run the eBPF probe is the following:
+
+- `CAP_SYS_ADMIN`
+- `CAP_SYS_RESOURCE`
+- `CAP_SYS_PTRACE`
+
+### Explanation
+
+- **CAP_SYS_RESOURCE**: Falco needs this to call the `setrlimit` syscall. This syscall is used together with `RLIMIT_MEMLOCK` flag to change the amount of memory that can be mlocked into RAM. The default memory limit value is very low, so even a very simple eBPF program will fail. The workaround is to increase the default value to something acceptable so that eBPF maps can be correctly mlocked in memory.
```

Comment:
```suggestion
- **CAP_SYS_RESOURCE**: Falco needs this capability to be able to call the `setrlimit` syscall. The `setrlimit` syscall is used together with the `RLIMIT_MEMLOCK` flag to change the amount of memory that can be _mlocked_ into RAM. The default value for this memory limit is very low, so even a very simple eBPF program would fail. The workaround is to increase the default value to something acceptable so eBPF maps can be correctly _mlocked_ in memory.
```

Reply:
so even a very simple eBPF program ... would fail or could fail?

Reply:
would fail, thank you!